smartsim/settings/builders/launch/alps.py

```diff
@@ -213,3 +218,13 @@ def set(self, key: str, value: str | None) -> None:
         if key in self._launch_args and key != self._launch_args[key]:
             logger.warning(f"Overwritting argument '{key}' with value '{value}'")
         self._launch_args[key] = value
+
+    def finalize(
```

Comment:
just for clarification, this is building something that the launcher, in this case shellLauncher can actually deploy?

Reply:
Exactly!! Now if for what ever reason some changes the input to the `ShellLauncher` mypy will automatically tell us that we need to update this class as well

Reply:
The finalize method has since been removed in favor of passing a function to the `dispatch` decorator. Marking resolved!